cliquet/resource/__init__.py

```diff
@@ -0,0 +1 @@
+from .base import *
```

Comment:
We should not import * there, either move the base code here or define the ``__all__``  variable.